tests/unit/test-firefox/test.remote.js

```diff
@@ -376,14 +376,38 @@ describe('firefox.remote', () => {
 
   });
 
-  describe('defaultRemotePortFinder', () => {
+  describe('findFreeTcpPort', async () => {
+
+    const port = await findFreeTcpPort();
+
+    function prepareConnection(portNumber = port, options = {}) {
```

Comment:
I am a little unsure about this change I made. I am not sure if ```prepareConnection``` was required here.